docs/source/using/addons.adoc

```diff
@@ -11,9 +11,12 @@ toc::[]
 [[add-ons-overview]]
 == Overview
 
-NOTE: This feature is still considered experimental and might change in future releases.
+[NOTE]
+====
+This feature is still considered experimental and might change in future releases.
+====
```

Comment:
Add-on is not an experimental feature anymore. So we should remove this note. cc @minishift/minishift-dev